kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -977,8 +983,20 @@ public LocalDate retrieve(String value) {
 
         return mapping;
     }
+    public void removeSpecialChar(ArrayList<String > listOfIds,Map<String, Object> record){
 
+        for(String id :listOfIds){
+            try{
+                String idValue=(String)record.get(id);
+                idValue=idValue.replaceAll("[\\n\\t\\r ]","");
+                record.replace(id,idValue);
+            }
+            catch (Exception e){
+                continue;
```

Comment:
Exception should never be eaten up. Please either log this or handle this.

Reply:
> Which exception is expected here? A more concrete Exception should be caught, and should never be eaten up. Please either log this or handle this.

Some of Ids may have null values(Not all ids are required in csv/xml records),  
So hasmap will throw an exception in that case and code will stopped.